cmake/Macros.cmake

```diff
@@ -66,9 +66,7 @@ macro(csfml_add_library target)
     # build dylibs
     if(SFML_OS_MACOSX AND BUILD_SHARED_LIBS)
         # adapt install directory to allow distributing dylibs in user’s application bundle
-        set_target_properties(${target} PROPERTIES
-                              BUILD_WITH_INSTALL_RPATH 1
```

Comment:
https://github.com/SFML/CSFML/pull/80

Here's the commit that added this. No rationale was provided in the PR thread so I'm not sure why this was needed. CMake's rpath support has evolved over the years so it's possible this was required 9 years ago but is not required anymore now that we're using CMake >3.22.

Reply:
We can always adjust when we find out what it was used for 😄